modules/ui/sources/window.es

```diff
@@ -121,6 +121,9 @@ export default class {
 
     }.bind(this));
 
+    //mock default FF function
+    this.popup.enableOneOffSearches = function() {}
```

Comment:
Hey guys, Drew from the Firefox team here. Mocking that method does seem like the best thing to do. If you're getting that error at startup, it's probably because the urlbar's constructor calls that method on the popup. It's just assumed that the popup implements it, even if one-off searches are disabled. The urlbar could maybe check if one-off searches are disabled before calling any one-off related methods, but I'm not sure that's a good idea or better than just mocking the method like you've done here.